src/layouts/ltrTreeLayout/index.js

```diff
@@ -23,7 +23,7 @@ class LTRTreeLayout {
   run (graph, dimensions, layoutComplete) {
     const layoutWorker = LayoutWorker();
     const layoutWorkerComplete = (event) => {
-      graph.nodes.forEach(node => {
+      graph.nodes.forEach((node) => {
```

Comment:
No idea what added this; maybe Emacs, maybe the linter. Let me know if I should remove it.